Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -191,7 +191,7 @@ class SkiaGLContext : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Ski
 
 GLContext* PlatformDisplay::skiaGLContext()
 {
-#if !(PLATFORM(PLAYSTATION) && USE(COORDINATED_GRAPHICS))
+#if PLATFORM(GTK) || PLATFORM(WPE) || PLATFORM(PLAYSTATION) && !USE(COORDINATED_GRAPHICS)
```

Comment:
I don't see why this change is necessary. The reason for the `#if !(PLATFORM(PLAYSTATION) && USE(COORDINATED_GRAPHICS))` was because PlayStation's OpenGL implementation doesn't support a context per thread. I get that ANGLE doesn't do that either but the Web Compositor was fine on PlayStation.